### PR TITLE
fix: js function getDom return null bug (#1374)

### PR DESCRIPTION
**Pull request by zxyao145**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#1374 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix  js function getDom return null bug   |
| 🇨🇳 Chinese |    修复技术中 getDom 函数可能返回 null 的bug      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by anddrzejb** [comment]
I am not sure about this PR 😢 . It seems to me that it is hiding a problem of incorrect `ElementReference` being passed to js and not really solving why there is an incorrect value. I was tracking a similar issue starting from [this comment](https://github.com/ant-design-blazor/ant-design-blazor/pull/1342#issuecomment-819237133).

File: components/core/JsInterop/JSInteropConstants.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public static class JSInteropConstants
 
         public static string Log => $"{FUNC_PREFIX}log";
 
+        [Obsolete("It will be removed in the future, because Blazor already has a native implementation.")]
```
[the remark this answers is not in the document]
Comment (zxyao145) [reply components/core/JsInterop/JSInteropConstants.cs:39]: Yes, you are right, and fixed.

**Comment by anddrzejb** [comment]
@zxyao145 Can you explain to me when you get the nulls from `getDom(element)`? Is it possible this happens when also `element` is null? Can I recreate this? As I wrote earlier, I have a feeling we are hiding a problem here, not solving it...

**Comment by zxyao145** [comment]
> @zxyao145 Can you explain to me when you get the nulls from `getDom(element)`? Is it possible this happens when also `element` is null? Can I recreate this? As I wrote earlier, I have a feeling we are hiding a problem here, not solving it...

Sorry, I don't know what happened. However, as issues 1374 said, when I quickly opened modal, I found that the `getDom` function (line 15) in JS would trigger this bug. In order to avoid this problem caused by other use of `getDom` function in JS, I added judgment to all use of `getDom` function.

**Comment by zxyao145** [comment]
> This is wasm or server side?

server side

**Review by anddrzejb — APPROVED** [review]
Sorry it took me so long. I got a different exception - in my case js was complaining about e. But your PR does fix this problem. I don't think there is any another simple solution for these fast-clicks operations - in server side it seems these type of problems will always be waiting to hit us. In wasm I don't think it will ever surface. I am afraid though, that this null handling may hide another issue in the future. Time will tell.

**Comment by zxyao145** [comment]
Sincerely thank you for all you've done for this PR.

**Review by ElderJames — APPROVED** [review]

**Pull request merged**